packages/mds-ingest-service/repository/index.ts

```diff
@@ -474,30 +474,25 @@ class IngestReadWriteRepository extends ReadWriteRepository {
 
   public getLatestTelemetryForDevices = async (device_ids: UUID[]): Promise<TelemetryDomainModel[]> => {
     try {
+      const connection = await this.connect('ro')
+
       /**
-       * https://github.com/typeorm/typeorm/blob/master/docs/transactions.md#using-queryrunner-to-create-and-control-state-of-single-database-connection
+       * Disable bitmapscan and run the query within a transaction
```

Comment:
Nit, but I'd invert this comment a bit
```suggestion
       * Run the query within a transaction, so we can safely disable bitmapscan
```

Reply:
Much clearer intent. Updated.